numpy/linalg/linalg.py

```diff
@@ -2610,12 +2610,12 @@ def norm(x, ord=None, axis=None, keepdims=False):
 
 # multi_dot
 
-def _multidot_dispatcher(arrays):
+def _multidot_dispatcher(arrays, out=None):
     return arrays
```

Comment:
Should this visit `out` too?

Reply:
Cc @shoyer.

Reply:
@eric-wieser @shoyer grepping around (`rg -A 4 "_dispatcher.*out"`) it seems that `out` should be returned by the dispatcher. Other functions with an optional `out` return it. Added in 5427885afdc09d198d914de13d321b37780176fb